### PR TITLE
Fix missing word in docblock

### PR DESCRIPTION
**Pull request by svenluijten**
the word "framework" was missing in the docblock above Vue in `resources/js/bootstrap.js`.

I managed to make the docblock fit Laravel's style by changing around some of the wording while keeping the general message the same.

### CODE REVIEW

**Comment by brunocascio** [comment]
Vuejs is not a framework. Is a library. :)

`Simple yet powerful library for building modern web interfaces. http://vuejs.org`

Docs: https://github.com/vuejs/vue

**Comment by svenluijten** [comment]
Damnit you're right. Let me fix that and change the wording around some
more.

**Comment by svenluijten** [comment]
What do you think about it now, @brunocascio? :smile:

**Comment by brunocascio** [comment]
Great @svenluijten :dancer:

File: resources/assets/js/bootstrap.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ window.$ = window.jQuery = require('jquery');
 require('bootstrap-sass/assets/javascripts/bootstrap');
 
 /**
- * Vue is a modern JavaScript for building interactive web interfaces using
- * reacting data binding and reusable components. Vue's API is clean and
- * simple, leaving you to focus only on building your next great idea.
+ * Vue is a modern JavaScript library for building interactive web interfaces
+ * using reactive data binding and reusable components. Vue's API is clean
+ * and simple, leaving you to focus on building your next great project.
```
Comment (GrahamCampbell) [inline resources/assets/js/bootstrap.js]: If we dropped the comma on this line, we'd have perfect alignment. 😆

Comment (svenluijten) [inline resources/assets/js/bootstrap.js]: I think that would decrease readability though, @GrahamCampbell :smile:

**Pull request merged**